src/cljdoc/config.clj

```diff
@@ -104,6 +104,9 @@
 (defn maven-repositories []
   (get-in (config) [:maven-repositories]))
 
+(defn extension-namespaces [config]
+  (clojure.core/get-in config [:extension-namespaces]))
```

Comment:
May be just `get-in`? Why do we use a fully qualified name here?

Reply:
The namespace defines it's own `get-in` which throws if a value is missing. It also excludes `clojure.core/get-in` from being referred alongside the rest of `clojure.core`:

```clj
(ns cljdoc.config
  (:refer-clojure :exclude [get-in])
```

Reply:
ah! understood. I should have read the code above. Thanks for clarification 🙂